austrakka/components/widget/funcs.py

```diff
@@ -0,0 +1,49 @@
+from os import path
+from austrakka.utils.helpers.output import call_get_and_print_table
+from austrakka.utils.api import api_post
+from austrakka.utils.api import api_put
+from austrakka.utils.misc import logger_wraps
+from austrakka.utils.paths import WIDGET_PATH
+
+
+@logger_wraps()
+def add_widget(name: str):
+    """
+    Add a widget for later inclusion in a dashboard.
+    """
+    api_post(
+        path=WIDGET_PATH,
+        data={
+            "name": name
+        }
+    )
+
+
+@logger_wraps()
+def update_widget(widget_id: int, new_name: str):
+    """
+    Update an existing widget.
+    """
+    api_put(
+        path=path.join(WIDGET_PATH, str(widget_id)),
```

Comment:
I think we should just use string interpolation with `/` to avoid issues with windows users